### DISCUSSION TITLE
Controlling subwoofer level?

### DISCUSSION THREAD

**Discussion by modo1302** [PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar#57] [Q&A]
Hi, is there a way to control subwoofer level via this integration?
(it's doable via the SmartThings app)
![Screenshot_20240105_133604_SmartThings](https://github.com/PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar/assets/133964285/02ad422d-0d62-4237-88eb-6edd6e60b60b)
![Screenshot_20240105_133608_SmartThings](https://github.com/PiotrMachowski/Home-Assistant-custom-components-SmartThings-Soundbar/assets/133964285/6c903d73-c2b7-469a-8b56-e181a5ff2fab)

**Comment by PiotrMachowski** [comment]
There is no documented way to do it via SmartThings API
